Server/tests/test_game.py

```diff
@@ -1,5 +1,8 @@
 import pytest
-from painttheworld.game import GameState, RED, BLUE
+from painttheworld.game import GameState
+from painttheworld.constants import RED, BLUE
```

Comment:
`from painttheworld.constants import Team`
and then you access colors like:
`Team.RED`